kafka-observer/Dockerfile.consumer

```diff
@@ -0,0 +1,21 @@
+FROM icr.io/codeengine/golang:latest AS stage
+
+WORKDIR /app/src
+
+COPY cmd/consumer/ ./consumer/
+
+COPY internal/ ./internal/
+
+COPY go.mod .
+
+COPY go.sum .
+
+RUN CGO_ENABLED=0 GOOS=linux go build -o consumer ./consumer
+
+FROM icr.io/codeengine/golang:latest
```

Comment:
```suggestion
FROM icr.io/codeengine/alpine
```

Using the alpine image reduces the image size from > 340MB to <10 MB. In consequence, the cold-start time of consumer jobs , which is a critical performance factor, reduces dramatically from >25s (due to image pull) to ~5s

Reply:
```
19m         Normal   Scheduled   pod/payments-consumer-jobrun-czj2n-0-0   Successfully assigned 18e4mb0tj2ml/payments-consumer-jobrun-czj2n-0-0 to 192.168.2.98
19m         Normal   Pulling     pod/payments-consumer-jobrun-czj2n-0-0   Pulling image "private.de.icr.io/ce--76586-18e4mb0tj2ml/job-payments-consumer"
19m         Normal   Pulled      pod/payments-consumer-jobrun-czj2n-0-0   Successfully pulled image "private.de.icr.io/ce--76586-18e4mb0tj2ml/job-payments-consumer" in 20.377397494s (20.377421762s including waiting)
19m         Normal   Created     pod/payments-consumer-jobrun-czj2n-0-0   Created container payments-consumer
19m         Normal   Started     pod/payments-consumer-jobrun-czj2n-0-0   Started container payments-consumer
```